composer.json

```diff
@@ -15,7 +16,10 @@
 			"app/database/migrations",
 			"app/database/seeds",
 			"app/tests/TestCase.php"
-		]
+		],
+		"psr-4": {
+			"models\\": "app/models"
```

Comment:
why not load whole source directory to own namespace. E.g. `"App\\": "app/"`
Now i got why your service provider in `model` namespace